.github/workflows/nv-inference.yml

```diff
@@ -40,7 +41,7 @@ jobs:
           git clone https://github.com/huggingface/transformers
           cd transformers
           # if needed switch to the last known good SHA until transformers@master is fixed
-          # git checkout 1cc453d33
+          git checkout v4.21.2
```

Comment:
I think we can change to this be latest master from HF, after we merge: https://github.com/microsoft/DeepSpeed/pull/2291. But I want to re-test #2291 after this PR is merged.